### PR TITLE
Map 20w09a blocks and items

### PR DESCRIPTION
**Pull request by Earthcomputer**
0w0bsidian

### CODE REVIEW

**Review by jaskarth — APPROVED** [review]

File: mappings/net/minecraft/advancement/criterion/TargetHitCriterion.mapping [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+	METHOD method_24861 trigger (Lnet/minecraft/class_3222;I)V
+		ARG 1 player
+		ARG 2 signalStrength
+	CLASS class_4852 Conditions
+		FIELD field_22430 signalStrength Lnet/minecraft/class_2096$class_2100;
+		METHOD <init> (Lnet/minecraft/class_2096$class_2100;)V
+			ARG 1 signalStrength
+		METHOD method_24864 test (I)Z
+			ARG 1 signalStrength
+		METHOD method_24865 crate (Lnet/minecraft/class_2096$class_2100;)Lnet/minecraft/class_4851$class_4852;
```
Comment (liach) [inline mappings/net/minecraft/advancement/criterion/TargetHitCriterion.mapping]:
```suggestion
		METHOD method_24865 create (Lnet/minecraft/class_2096$class_2100;)Lnet/minecraft/class_4851$class_4852;
```

File: mappings/net/minecraft/item/Items.mapping [hunk 2]
```diff
@@ -148,6 +148,8 @@ CLASS net/minecraft/class_1802 net/minecraft/item/Items
 	FIELD field_22019 ANCIENT_DEBRIS Lnet/minecraft/class_1792;
 	FIELD field_22031 CRIMSON_PLANKS Lnet/minecraft/class_1792;
 	FIELD field_22032 WARPED_PLANKS Lnet/minecraft/class_1792;
+	FIELD field_22420 TARGET Lnet/minecraft/class_1792;
```
Comment (liach) [inline mappings/net/minecraft/item/Items.mapping:151]: Is the item id just `minecraft:target`?

**Review by liach — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/item/Items.mapping (hunk 2 above)
Comment (Shnupbups) [reply mappings/net/minecraft/item/Items.mapping:151]: Yes, it is

**Review by Shnupbups — CHANGES_REQUESTED** [review]
Looks good, beside the typo liach pointed out in `TargetHitCriterion`

**Review by liach — APPROVED** [review]

**Review by Runemoro — APPROVED** [review]

**Review by Shnupbups — APPROVED** [review]

**Pull request merged**